charts/simple-workflow/templates/workflow-pipeline.yaml

```diff
@@ -176,6 +193,16 @@ spec:
               {{- else }}
               value: status.sync.status == Synced, status.health.status == Healthy, status.sync.comparedTo.source.targetRevision == {{ $.Values.app.targetRevision }}
               {{- end }}
+       {{- range $jobName, $jobConfig := $config.jobs }}
+        - name: verify-job-synced-and-healthy-{{ $name }}
+          depends: verify-app-synced-and-healthy-{{ $name }}.Succeeded
+          template: verify-job-{{ $jobName }}
+          arguments:
+            parameters:
+              - name: successCondition
+                value: status.health.status == Healthy
```

Comment:
Do we need both synced and healthy?

Reply:
In case of a job there is no Sync actually.

<img width="621" alt="Screenshot 2023-08-16 at 3 36 10 PM" src="https://github.com/Nextdoor/k8s-charts/assets/73497410/b6880864-909e-46d9-b0a2-83fa91f52528">